collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	MNT_RDONLY            = 0x1
```

Comment:
The idea was to keep it similar to the actual constants in the header files for each OS, since although BSD and Linux use the same ones they do call them different things (and syscall unfortunately defines neither).